.circleci/config.yml

```diff
@@ -185,8 +184,10 @@ commands:
           tar -zxvf /tmp/helm.tar.gz -C /tmp
           sudo mv -f /tmp/linux-amd64/helm /usr/bin/helm
           chmod a+x /usr/bin/helm
-          sudo mv -f /tmp/linux-amd64/tiller /usr/bin/tiller
-          chmod a+x /usr/bin/tiller
+          if [[ -f /tmp/linux-amd64/tiller ]]; then
```

Comment:
Why are we installing tiller if it was removed from Helm 3?

Reply:
Is this so that we can run both Helm 2 and 3 tests?

Reply:
> Is this so that we can run both Helm 2 and 3 tests?

I left it in just for posterity, but it's actually not being used by pytest anymore since we're running helm from docker images now in the test.

Reply:
Ok then get rid of it.  Git is our posterity.